Bottom: 3aa54f96e45ed959080bfb86260c472d3c65ba95
Top:    93a24c8a50a3d76771bf74041003a24389724f2a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-17 11:03:35 -0500

fixup! Move ansible role vars to role defaults


---
diff --git a/roles/docker-mediawiki/defaults/main.yml b/roles/docker-mediawiki/defaults/main.yml
index 36f6bd3e0..93ab0c201 100644
--- a/roles/docker-mediawiki/defaults/main.yml
+++ b/roles/docker-mediawiki/defaults/main.yml
@@ -14,10 +14,10 @@ wiki_db_user: novatech
 # wiki_db_root_password: ''
 # wiki_db_password: ''
 
-wiki_image_repo: https://github.com/novatechweb/docker-mediawiki.git
-
 # the name of the image being duilt and used for the container
+wiki_image_repo: https://github.com/novatechweb/docker-mediawiki.git
 wiki_image_name: '{{ docker_registry_username }}/wiki'
+
 wiki_db_image_name: 'mysql:5'
 
 # the name of the container being started
